src/execute.rs

```diff
@@ -0,0 +1,35 @@
+use volta_core::error::report_error;
+use volta_core::log::{LogContext, LogVerbosity, Logger};
+use volta_core::run::execute_tool;
+use volta_core::session::{ActivityKind, Session};
+use volta_core::signal::setup_signal_handler;
+use volta_fail::ExitCode;
+
+#[inline]
```

Comment:
Why do we need this to be explicitly inlined? Can we add an explanatory comment so it's obvious to other later readers of the code? ❤️

Reply:
We probably don't, and it's likely optimized by the compiler anyway since the two binaries that call this are single function calls with nothing else anyway. I'll remove it :+1: